doc/cifarSC2011/gpundarray.txt

```diff
@@ -24,7 +24,7 @@ Why a common GPU ndarray?
 
 Design Goals
 
-- Make it VERY similar to ``numpy.ndarray``
```

Comment:
same here.
Actually, these are old tutorials that got archived here, it is not really necessary to update them.